facenet/fr_utils.py

```diff
@@ -0,0 +1,201 @@
+#### PART OF THIS CODE IS USING CODE FROM VICTOR SY WANG: https://github.com/iwantooxxoox/Keras-OpenFace/blob/master/utils.py ####
+
+import tensorflow as tf
```

Comment:
please add docstring for what this file does  - the name is too generic for the file, worth changing it to be more meaningful as well

Reply:
ok

Reply:
I have put all the description can you plz check it from file heading